src/components/profile/TableNotifications.vue

```diff
@@ -0,0 +1,184 @@
+<script lang="ts">
+/**
+ * TableNotifications Component
+ *
+ * @description * Use this component to display notifications in a table.
+ *
+ * Used in `ProfileTabs`
+ *
+ * @components
+ * - `CHILD`: Component to ... .
+ *
+ * @example
+ * <table-notifications />
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=4858-105372&t=WcrxMvLONggUrjGt-1)
+ */
+
+// libraries
+import { date } from 'quasar';
+import { computed, defineComponent, reactive } from 'vue';
+
+// composables
+import { i18n } from '../../boot/i18n';
+
+// fixtures
+import tableNotificationsFixture from '../../../test/cypress/fixtures/tableNotifications.json';
+
+// types
+import type { TableColumn } from '../types/Table';
+import type { Notification } from '../types/Notifications';
+
+export default defineComponent({
+  name: 'TableNotifications',
+  setup() {
+    const columns: TableColumn[] = [
+      {
+        name: 'title',
```

Comment:
`name` key value string literal should be TS Enum.

Reply:
Refactored into enum.